### PR TITLE
implemented user profile picture update and delete

### PR DESCRIPTION
**Pull request by OladapoAjala**
### What this PR does (required):
- This PR resolves issue #26
- It implements APIs for uploading & deleting user profile pictures

### Screenshots / Videos (front-end only):
Screenshot from 2021-11-02 14-17-28
Screenshot from 2021-11-02 14-25-22

### Any information needed to test this feature (required):
1. Create a new mongodb cluster and paste your connection URI in the ".env" file
2. Set a random JWT secret
3. Set CLOUDINARY_URL in the ".env" file
4. Start the server by running "npm run dev"
5. Use postman to test out the available end-points
6. Upload image => POST   http://127.0.0.1/api/v1/users/upload-image
7. Delete image =>  DELETE   http://127.0.0.1/api/v1/users/delete-image

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -21,3 +26,50 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+// @route POST /users/upload-image
+// @desc upload user profile picture
+// @access Private
```
Comment (ezeilo-su) [inline server/controllers/user.js:32]: Good documentation

File: server/controllers/user.js [hunk 2]
```diff
@@ -21,3 +26,50 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
...
+  res.status(200).json({
+    success: {
+      user,
+    },
+  });
+});
+
+// @route DELETE /users
+// @desc delete user profile picture
+// @access Private
```
Comment (ezeilo-su) [inline server/controllers/user.js:60]: :+1:

**Review by ezeilo-su — COMMENTED** [review]
Looks nice :+1:

File: server/controllers/user.js (hunk 1 above)
Comment (OladapoAjala) [reply server/controllers/user.js:32]: Thank you @ezeilo-su

**Review by moffatethan — APPROVED** [review]

**Pull request merged**